.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Release
+
+on:
+  push:
+    tags: [ '**' ]
+  workflow_dispatch:
+
+concurrency: release-${{ github.ref }}
+
+env:
+  IMAGE_NAME: hri-flink-pipeline-core
+
+jobs:
+  buildAndRelease:
+    name: Gradle Build and Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set branch name
+        uses: nelonoel/branch-name@v1.0.1
+
+      - uses: actions/checkout@v2
+
+      - name: Set up adoptopenjdk8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+          cache: 'gradle'
+
```

Comment:
Does this action code correctly set the tags for releases?  I feel like perhaps we are missing some operation here.  For example, I think we need something to set the `ACTIONS_TAG` Env Var.

Reply:
I agree. Either the tag needs to be extracted here, or the build.gradle logic needs to be updated.

Reply:
I added the tag extraction ✅